locales/base/translation.json

```diff
@@ -1292,6 +1292,33 @@
     "disclaimer": "You acknowledge and agree that your Personal Data will be provided to us and shared with third-parties for verification purposes as stated in the <0>Privacy Policy</0>",
     "button": "Set up ID Verification"
   },
+  "fiatConnectKycStatusScreen": {
+    "contactSupportPrefill": "I'm having trouble verifying my identity with Valora.",
```

Comment:
@jh2oman/laura/nitya, any idea what this prefill text should be? I just guessed here, but wondering if there's something specific it should be instead? (Also, should it vary for different KYC states?)

Reply:
I'd vote for including the KYC state in the message somehow. sounds potentially useful for debugging